roles.md

```diff
@@ -2,8 +2,125 @@
+
+## Project Steering Committee
+
+The Project Steering Committee (PSC) is the official managing body for our project. The committee is drawn from the community and is based on merit and interest.
+
+*   We aim to attract a diverse mix of personal characteristics, reflective of the community we represent.
+*   We wish to avoid having one organisation employing a controlling proportion of the PSC.
+
+A PSC chair shall be selected from the PSC, with the only official extra duty being that they have a deciding vote in the case of a tie.
+
+**Access:**
+
+*   PSC members may be granted admin access to our github repositories if they have a need for them.
+
+**Additional expectations of PSC members:**
+
+*   Consider opinions from within our community, and advocate for the best interests of our community when making decisions.
+*   Vote according to your personal best judgment, as opposed to voting on behalf of your employer.
+*   Vote promptly (usually within days) to motions being raised.
+*   Regularly participate in project activities.
+*   Help with whatever unglamourous work that is required to keep the project moving forward.
+*   Monitor conversations within our forums and contribute constructively where appropriate.
+*   Optionally attend weekly meetings.
+
+**Eligibility:**
+
+Before being accepted for this role, a person should display the following characteristics:
+
+*   Sustained track record displaying the characteristics expected of PSC members, typically while acting as a committer within our project, typically for six months or more.
+*   Express an interest in ongoing participation.
+*   Acknowledge the PSC member expectations.
+
+## Mentors
+
+Initiatives which are funded by The Good Docs Project should have a mentor.  
+
+**Expectations of mentors:**
+
+A mentor’s responsibilities typically includes:
+
+*   Mentor a worker in developing a proposal.
+*   Facilitate making connections with other members within the community.
+*   Attend weekly meetings and provide guidance where applicable.
+*   If needed, escalate concerns to the Project Steering Committee.
+*   Review work done, and recommend the approval or rejection of the project based on work done.
+
+**Eligibility:**
+
+A mentor will typically be a member of the PSC, but may be an experienced person who is trusted by the PSC.
+
+Any conflict-of-interest the mentor may have should be stated beforehand. This could result from mentoring managers, direct reports, relatives, or close friends. The PSC will need to decide whether the conflict will negatively impact the mentoring experience.
+
+## Treasurers
+
+Treasurers are PSC members who additionally have been provided with admin access to our [financial account with Open Collective](https://opencollective.com/thegooddocsproject).
+
+**Expectations of treasurers:**
+
+*   Notify the PSC of Open Collective funding requests by [raising a motion](decisions.html).
+*   AFTER receiving approval from the PSC, approve funding requests.
```

Comment:
Responsible financial management?

Reply:
Put that in as an alpha cut, then plan to refine the sentence/guidance in Alpha 0.1 or 0.2?